.github/workflows/pr.yml

```diff
@@ -0,0 +1,16 @@
+name: Pull Request Validator
+
+on: pull_request
+
+jobs:
+  message-check:
+    if: "!contains(github.event.pull_request.title, '[merge]')"
```

Comment:
Hi @kpodemski 🙂 
Consider this cases : 

1 - PR Title  = **Use [merge] channel in config.yml** ([config.yml](https://github.com/PrestaShop/PrestaShop/blob/develop/app/config/config.yml#L63))
 2 - PR Title = **Use array merge instead of array combine**

What's going to happen ? Will the PR be blocked or not?

Maybe use **startsWith** instead of **contains** ?

Reply:
1. this would be blocked, is this even a valid setting? I thought about possible scenarios and didn't came up with anything 😅 
2. this would be allowed

> Maybe use startsWith instead of contains ?

this would solve ad1 if that's a real issue